BlueBrain/nexus-web
# Feature 3224 // studio data download

Fixes #[3224](https://app.zenhub.com/workspaces/blue-brain-nexus-60ace035034cae00105c9307/issues/gh/bluebrain/nexus/3224)
<img width="1018" alt="Screenshot 2023-05-11 at 19 24 48" src="https://github.com/BlueBrain/nexus-web/assets/11242410/a905ba55-ae0a-41f4-a7c2-693cf05ee3b4">


## Description

Enable users to download metadata and files associated with rows in studios table.

## How to test
1. Open a dashboard in studios
2. Select a row to download
3. Click on download button in datapanel -> A tar file should be downloaded.

## Types of changes


## Checklist:
